scripts/aggregate_build_tests.py

```
@@ -35,7 +38,61 @@ def list_build_files(date: int) -> List[str]:
+            arch = f"{arch}-malloc"
+
+        os = file.split("/")[3].replace(
+            "-malloc", ""
+        ).replace(
+            "-aarch64", ""
+        ).replace(
+            "-relwithdebinfo",
+            ""
+        )
+
+    return os, arch
```

Pylint (3.3.7)

[error] Possibly using variable 'os' before assignment

(E0606)